admin/sql/updates/schema-change/28.all.sql

```diff
@@ -31,4 +32,17 @@ $$ LANGUAGE 'plpgsql';
 
 CREATE INDEX IF NOT EXISTS unreferenced_row_log_idx_inserted ON unreferenced_row_log USING BRIN (inserted);
 
+--------------------------------------------------------------------------------
+SELECT '20221114-mbs-12704.sql';
+
+
+DROP TRIGGER IF EXISTS a_ins_editor ON editor;
+
+DROP FUNCTION a_ins_editor();
+
+DROP TABLE editor_watch_artist;
+DROP TABLE editor_watch_preferences;
+DROP TABLE editor_watch_release_group_type;
+DROP TABLE editor_watch_release_status;
+
```

Comment:
It appropriately targets all instances and avoids failing if the trigger doesn't exist (as in mirrors).